bedrock/products/templates/products/vpn/resource-center/base-article.html

```diff
@@ -0,0 +1,87 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{% block article_title %}{% endblock article_title %}{% endblock page_title%}
+
+{% block content %}
+<nav class="mzp-c-breadcrumb">
+  <ol class="mzp-c-breadcrumb-list">
+    <li class="mzp-c-breadcrumb-item">
+      <a href="{{ url('products.vpn.resource-center.landing') }}">{{ ftl('vpn-resource-center-title') }}</a>
+    </li>
+    <li class="mzp-c-breadcrumb-item">
+      {{ self.article_title() }}
+    </li>
+  </ol>
+</nav>
+<div class="vpn-article-container">
+  <div class="mzp-l-content mzp-t-content-md">
+      <article class="mzp-c-article vpn-article">
+        {# Categories will be used at some point but not yet #}
+        {# <p class="vpn-article-category">{% block article_category %}{{ self.article_category().capitalize() }}{% endblock %}</p> #}
+        <h2>{{ self.article_title() }}</h2>
+        {% block vpn_article %}{% endblock vpn_article %}
+      </article>
+    <div class="vpn-c-vote no-js">
+      <h3>{{ ftl('vpn-resource-center-was-this-article') }}</h3>
+      <div class="vpn-c-vote-btn-container">
+        <button type="button" data-cta-type="button" data-cta-text="{% block article_slug %}{% endblock %}" class="vpn-c-vote-btn up">
```

Comment:
```suggestion
        <button type="button" data-cta-type="button" data-cta-text="{% block article_slug %}{% endblock %}" class="vpn-c-vote-btn up">
        <span class="visually-hidden">Yes</span>
```

Reply:
@alexgibson good point about sr-accessible text. @nathan-barrett I think we can keep the svg and add a [visually-hidden](https://github.com/mozilla/protocol/blob/557f04e6252ee327760b2111d03d88a8a53d6f30/src/assets/sass/protocol/includes/_mixins.scss#L137) element inside the button to provide this infomation

Reply:
(as noted below, you need to use the `@visually-hidden` mixin in styling to get the behaviour, `class="visually-hidden" is an explicit selector to target, but will not work on its own)